src/globalStyles.ts

```diff
@@ -2,9 +2,24 @@ import { css } from '@emotion/react'
 import '@radix-ui/themes/styles.css'
 import 'allotment/dist/style.css'
 
+import InterVariable from '@/assets/fonts/Inter/InterVariable.woff2'
+
 export const globalStyles = css`
+  @font-face {
+    font-family: 'InterVariable';
+    src: url(${InterVariable}) format('woff2');
+    font-weight: 400 500 600 700;
```

Comment:
Figured I'd use the variable font, as adding new weights and styles will be easier (and it has [lots of interesting features](https://rsms.me/inter/#features))